docs/index.rst

```diff
@@ -0,0 +1,3 @@
+######
+PyLeco
```

Comment:
Add some point we should decide on, and harmonize, the capitalization -- PyLECO, PyLeco, pyleco,...

Reply:
We could start right at this moment. Maybe in a full issue.

Reply:
I changed it everywhere to PyLECO

Reply:
Thanks!